lib/ueberauth/strategy/facebook.ex

```diff
@@ -150,26 +157,13 @@ defmodule Ueberauth.Strategy.Facebook do
     end
   end
 
-  defp user_query(conn, token) do
-    %{"appsecret_proof" => appsecret_proof(token)}
```

Comment:
Why remove the appsecret_proof? This seems like it could make requests more unreliable.

https://developers.facebook.com/docs/graph-api/securing-requests#appsecret_proof